src/lib.rs

```diff
@@ -102,10 +103,26 @@
 //!
 //! [`Slab::with_capacity`]: struct.Slab.html#with_capacity
 
+#[cfg(not(feature = "std"))]
+extern crate alloc;
+
+#[cfg(not(feature = "std"))]
+use alloc::vec::Vec;
+
+#[cfg(not(feature = "std"))]
+use alloc::vec;
+
+#[cfg(not(feature = "std"))]
+use core::iter::{FromIterator, IntoIterator};
```

Comment:
```suggestion
use core::iter::FromIterator;
```

`IntoIterator` is included in the prelude.